services/drupal/composer.json

```diff
@@ -231,7 +231,7 @@
         "drupal/config_filter": "^1.8",
         "drupal/config_ignore": "^2.3",
         "drupal/config_split": "~1.5",
-        "drupal/content_moderation_notifications": "dev-3.x#7ffa936",
+        "drupal/content_moderation_notifications": "dev-3.x",
```

Comment:
I typically prefer pinning the commit in the dependency specification when tracking dev branches to avoid accidental updates to newer untested versions of the software.  So if we need to update to a newer hash let's bake that in here.